features/commands.js

```diff
@@ -213,6 +213,8 @@ You can paste small bits of code directly in chat with syntax highlighting:
 \\\`\\\`\\\`
 
 Or you can upload larger blocks of code (or files) to a service such as https://gist.github.com.
+
+We are also big fans of https://codesandbox.io where you can set up a working example of your code (or reproduce your bug). You can even link it to an existing GitHub repo so that others can try out your code live and help you solve your problems.
```

Comment:
Maybe we should link directly to https://codesandbox.io/s/new to ease the process?